src/libponyc/expr/call.c

```diff
@@ -238,7 +238,17 @@ static bool check_arg_types(pass_opt_t* opt, ast_t* params, ast_t* positional,
     ast_t* wp_type = consume_type(p_type, TK_NONE);
     errorframe_t info = NULL;
 
-    if(!is_subtype(arg_type, wp_type, &info, opt))
+    if(NULL == wp_type)
```

Comment:
this should be reversed. 

i'm looking at the test failures now so if i get those figured out, i'll also do this.

Reply:
Thanks, sorry for dropping the ball on this, I've had a lot going on in life right now.